test/Microsoft.TemplateEngine.Cli.UnitTests/Microsoft.TemplateEngine.Cli.UnitTests.csproj

```diff
@@ -10,6 +10,8 @@
 
   <ItemGroup>
     <PackageReference Include="FakeItEasy" />
+    <PackageReference Include="Verify.XUnit" Version="14.2.0" />
```

Comment:
I assume this is already compatible with SourceBuild and we won't have problems with this, right?

Reply:
Tests are excluded from source build, best to my knowledge.

Reply:
tests are not enabled in source build